src/main/webapp/IFrame.js

```diff
@@ -109,7 +111,7 @@ function onPlayerReady() {
 // Move playhead slightly ahead of updated timestamp when needed
 function seek(vidData) {
   if (vidData.isPlaying) {
-    const seekAhead = vidData.timestamp + SYNC_WINDOW*0.75;
+    const seekAhead = vidData.timestamp + SYNC_WINDOW * 0.85;
```

Comment:
You could move the 0.85 to a constant and tune it by changing it in one place instead of multiple changes which you might miss sometimes.

Reply:
Alright, that sounds good.